modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getYearlyTerms.js

```diff
@@ -4,52 +4,69 @@
  */
 
 const getYearlyTerms = ({endDate, startDate}) => {
-	const yearStartDate = new Date(startDate).getFullYear();
-	const yearEndDate = new Date(endDate).getFullYear();
+	const subscriptionStartDate = new Date(startDate);
```

Comment:
given that we have abstracted out this login into yearly terms, the variable name subscriptionStartDate does not seem to fit. I would keep it simpler. What type of variable is being passed into this function? we may need to rename a few things for clarity. For example, if it i a string being passed in, the the params should be endDateString and then you can convert that to endDate. I am happy to meet and discuss further if its helpful

Reply:
I've sent the adjustments, thanks!